utils/lj-releng

```diff
@@ -0,0 +1,228 @@
+#!/usr/bin/env perl
```

Comment:
the original source: https://github.com/openresty/openresty-devel-utils/blob/master/lj-releng

we should add the LICENSE information in this file:

https://github.com/openresty/openresty-devel-utils/tree/master?tab=readme-ov-file#copyright--license

Reply:
added